.vib/argo-cd/goss/argo-cd.yaml

```
@@ -0,0 +1,16 @@
+command:
+  check-app-version:
+    # argocd version exits with exit code 1
+    exec: {{ .Vars.version.bin_name }} {{ .Vars.version.flag }}
```

We don't usually reuse the vars used for the Goss' templates. In this case, for simplicity's sake, I would just have it here as:

```suggestion
    exec: argocd version
```